Classes/Managers/PlayKitManager.swift

```diff
@@ -48,6 +48,9 @@ import UIKit
     }
     
     @objc public func registerPlugin(_ pluginClass: BasePlugin.Type) {
+        if let pluginWarmUp = pluginClass as? PKPluginWarmUp.Type {
```

Comment:
@gal-orlanczyk the change is necessary - it is best practice to separate PR to specific change for tracking and being able to revert.
let's leave it for now but keep in mind for future PR.